grc/core/blocks/dummy.py

```diff
@@ -19,6 +19,7 @@
 
 from . import Block, register_build_in
 
+from ._build import _build_params, _build_ports
```

Comment:
lets not import protected functions from another module. If these are required, they'll have to be renamed

Reply:
Might be worth noting that I did the same when fixing the virtual blocks. I can rename them :+1: 

https://github.com/gnuradio/gnuradio/blob/df4e7a85263eb523395ce6bef0b2e7769bd63465/grc/core/blocks/virtual.py#L23

Reply:
As I'm new to python I indeed had a look how other blocks handle this. In this case it was @haakov 's virtual block.

Renaming the function is no problem.